Code/test/lib/agency-entity-fixture2.js

```diff
@@ -0,0 +1,35 @@
+/* jshint node: true */
```

Comment:
dont really need this.... could just create an entity fixture and then modify some of the attributes to make it different than the other. Avoid adding extra pages of code.